### ISSUE TITLE
fix streaming metric behavior

### ISSUE THREAD

**Issue by vervacity** [kundajelab/tronn#3]
Streaming metrics documentation [here](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/g3doc/api_docs/python/contrib.metrics.md)

Essentially the problem is that metrics are aggregated across all batches. This is good when you want to build a metric across batches (for example, getting metrics on full validation set at the end of an epoch). This is bad once you hit two epochs (because the metric does not reset for the second epoch). Need to find a fix or just calculate metrics separately from streaming, and just use streaming as a way to make sure the loss is dropping.

NOTE: I already calculate my own metrics separately, it's just a question of how accurate will tensorboard look when using streaming metrics
